searchcore/src/vespa/searchcore/proton/matching/match_thread.cpp

```diff
@@ -267,7 +268,8 @@ MatchThread::findMatches(MatchTools &tools)
             tools.setup_second_phase();
             DocidRange docid_range = scheduler.total_span(thread_id);
             tools.search().initRange(docid_range.begin, docid_range.end);
-            auto sorted_hit_seq = matchToolsFactory.should_diversify()
+            const MatchToolsFactory & mtf = matchToolsFactory;
```

Comment:
this capture seems unneeded. Consider renaming the long name instead, if it does not please you.

Reply:
Fixed